en/news/_posts/2019-03-13-ruby-2-5-4-released.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: news_post
+title: "Ruby 2.5.4 Released"
+author: "nagachika"
+translator:
+date: 2019-03-13 11:30:00 +0000
+lang: en
+---
+
+Ruby 2.5.4 has been released.
+
+This release includes some bug fixes and update of bundled rubygems which contains some secirity fiexes.
```

Comment:
same issues with https://github.com/ruby/www.ruby-lang.org/pull/2002

Reply:
I'll rebase to master before merge to fix this